### PR TITLE
Update protobufs

### PR DESCRIPTION
**Pull request by xPaw**
Also updated gc.proto and EMsg.

### CODE REVIEW

**Review by psychonic — APPROVED** [review]

File: Resources/NetHookAnalyzer2/NetHookAnalyzer2/Generated/.gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+*.cs text eol=crlf -diff
```
Comment (yaakov-h) [inline Resources/NetHookAnalyzer2/NetHookAnalyzer2/Generated/.gitattributes:1]: why not let autocrlf take care of it?

Comment (xPaw) [reply Resources/NetHookAnalyzer2/NetHookAnalyzer2/Generated/.gitattributes:1]: It was mostly for -diff, and I copied it from steamkits generated folder.

Comment (yaakov-h) [reply Resources/NetHookAnalyzer2/NetHookAnalyzer2/Generated/.gitattributes:1]: why not diff some nice text files?

FWIW GitHub seems to be ignoring it anyway

Comment (xPaw) [reply Resources/NetHookAnalyzer2/NetHookAnalyzer2/Generated/.gitattributes]: I removed it.

**Review by yaakov-h — APPROVED** [review]

**Pull request merged**